diff -Naur sox-14.3.2.orig/configure.ac sox-14.3.2/configure.ac
--- sox-14.3.2.orig/configure.ac	2012-03-22 10:00:31.150631401 -0500
+++ sox-14.3.2/configure.ac	2012-03-22 10:03:05.872649536 -0500
@@ -600,7 +600,10 @@
 if test "$using_lpc10" = yes; then
     lpc10_makefile=lpc10/Makefile
 fi
-AC_CONFIG_FILES(Makefile src/Makefile libgsm/Makefile $lpc10_makefile msvc9/Makefile msvc10/Makefile sox.pc)
+if test "$using_libgsm" = yes; then
+    libgsm_makefile=libgsm/Makefile
+fi
+AC_CONFIG_FILES(Makefile src/Makefile $libgsm_makefile $lpc10_makefile msvc9/Makefile msvc10/Makefile sox.pc)
 AC_OUTPUT
 
 if test "$using_gsm" != "no"; then
